runbotics/runbotics-desktop/src/action/general/general.action-handler.ts

```diff
@@ -44,6 +44,7 @@ export default class GeneralActionHandler extends StatelessActionHandler {
     async startProcess(
         request: DesktopRunRequest<GeneralAction.START_PROCESS, StartProcessActionInput>
     ): Promise<StartProcessActionOutput> {
+        // eslint-disable-next-line no-async-promise-executor
```

Comment:
You sure? The rejection is handled partially. What if you'd use 'await' and then/catch and returned the value instead of new Promise?

Reply:
I think I've found a compromise to solve this problem